.ci/openshift-ci/README.md

```diff
@@ -0,0 +1,63 @@
+
+The ci-operator has this [jobs directory](https://github.com/openshift/release/tree/master/ci-operator/jobs) on which Prow job files are placed on also per-component/repository/branch basis. Find the kata-containers' jobs configuration [here](https://github.com/openshift/release/tree/master/ci-operator/jobs/kata-containers). These files can be automatically generated from the ci-operator configuration files as follows:
+
+```
+$ git clone https://github.com/openshift/release
+$ cd release
+$ make jobs
+```
+
+# The CI pipeline explained
+
+The openshift-ci CI pipeline implemented to test kata-containers can be logically broken into the following stages:
+
+  * [Build container images](#build-container-images)
+  * [Create the test cluster](#create-the-test-cluster)
+  * [Install kata-containers](#install-kata-containers)
+  * [Run tests](#run-tests)
+
+## Build container images
+
+In this initial stage container images are prepared to be used along the pipeline. For instance, the base openshift image version or the build root image can be chosen. The preparation instructions are declared in the kata-containers's ci-operator configuration file.
+
+The following images are built (orderly):
+
+1. `pipeline:src` - wraps the source repository
+    * The source repository is copied to `$KATA_SRC/<repo>` in the image. Assume that `$KATA_SRC` is `/go/github.com/src/kata-containers`
+    * The repository and branch are inferred by the ci-operator unless explicitly declared
+2. `pipeline:bin` - wraps the built-from-source artifacts
+    * Uses the source repository packaged in `pipeline:src`
+    * The `$KATA_SRC/<repo>/.ci/openshift-ci/build.sh` script is called to build and install kata in the image, and the destination directory is `$KATA_SRC/<repo>/_out/build_install`
+3. `pipeline:kata-installer` - wraps the installation scripts and payload
+    * This image is built from the `$KATA_SRC/<repo>/.ci/openshift-ci/images/` Dockerfile context directory
+    * The ci-operator is instructed to map the kata installed directory from `pipeline:bin` to `_out` in that context directory. This constitutes the packaged kata-containers artifacts which will be installed in the test cluster.
+ 
+## Create the test cluster
+
+By the means of [templates](https://github.com/openshift/release/tree/master/ci-operator/templates) the ci-operator allows for preparing the test environment. Commonly those templates are made to instruct the operator on how to deploy and configure OpenShift in a given cloud provider, then run end-to-end tests.
+
+In special for kata-containers the used template needs to enable nested virtualization in the test cluster. See the template currently used in the kata-containers's ci-operator configuration file.
```

Comment:
The project should always be referred to as "Kata Containers" in documentation. But here I think you are referring to the directory? In which case, put it in back-ticks: `kata-containers/`.

Reply:
I re-wrote the README to use the project's name correctly as "Kata Containers".